File: twoPointers/26_remove_duplicates_from_sorted_array.go
// O(n) time | O(1) space - where n is nums.length
func removeDuplicates(nums []int) int {
    insertIdx := 1
    for idx, _ := range nums {
        if idx == 0 {continue}
        if nums[idx] != nums[idx - 1] {
            nums[insertIdx] = nums[idx]
            insertIdx++
        }
    }
    return insertIdx
}

File: twoPointers/26_remove_duplicates_from_sorted_array.py
class Solution:
    # O(n) time | O(1) space - where n is nums.length
    def removeDuplicates(self, nums: List[int]) -> int:
        insertIdx = 1
        for i in range(1, len(nums)):
            if nums[i] != nums[i - 1]:
                nums[insertIdx] = nums[i]
                insertIdx += 1
        
        return insertIdx

